tests/test_api.py

```diff
@@ -154,8 +154,8 @@ def test_serve_oauth_token(self):
         # TODO: login
         # TODO: get token
```

Comment:
at a later time perhaps perform something like dss/smoketest, where the cli is downloaded + used to login,logout

Reply:
I'll do something to that affect.